pkg/cache/internal/informers.go

```diff
@@ -444,6 +446,9 @@ func (ip *Informers) makeListWatcher(gvk schema.GroupVersionKind, obj runtime.Ob
 			},
 			// Setup the watch function
 			WatchFunc: func(opts metav1.ListOptions) (watch.Interface, error) {
+				opts.Watch = true // Watch needs to be set to true separately
```

Comment:
Why is this needed now?

Reply:
I just added it because I noticed it was in the other one, not sure if it is actually needed?